polygon/rest/models/snapshot.py

```diff
@@ -277,3 +277,145 @@ def from_dict(d):
             spread=d.get("spread", None),
             updated=d.get("updated", None),
         )
+
+
+@modelclass
+class UniversalSnapshotSession:
+    """Contains data about the most recent trading session for an asset."""
+
+    price: Optional[float] = None
+    change: Optional[float] = None
+    change_percent: Optional[float] = None
+    early_trading_change: Optional[float] = None
+    early_trading_change_percent: Optional[float] = None
+    late_trading_change: Optional[float] = None
+    late_trading_change_percent: Optional[float] = None
+    open: Optional[float] = None
+    close: Optional[float] = None
+    high: Optional[float] = None
+    low: Optional[float] = None
+    previous_close: Optional[float] = None
+    volume: Optional[float] = None
+
+    @staticmethod
+    def from_dict(d):
+        return UniversalSnapshotSession(**d)
+
+
+@modelclass
+class UniversalSnapshotLastQuote:
+    """Contains the most recent quote for an asset."""
+
+    ask: Optional[float] = None
+    ask_size: Optional[float] = None
+    bid: Optional[float] = None
+    bid_size: Optional[float] = None
+    midpoint: Optional[float] = None
+    exchange: Optional[int] = None
+    timeframe: Optional[str] = None
+    last_updated: Optional[int] = None
+
+    @staticmethod
+    def from_dict(d):
+        return UniversalSnapshotLastQuote(**d)
+
+
+@modelclass
+class UniversalSnapshotLastTrade:
+    """Contains the most recent trade for an asset."""
+
+    id: Optional[int] = None
+    price: Optional[float] = None
+    size: Optional[int] = None
+    exchange: Optional[int] = None
+    conditions: Optional[List[int]] = None
+    timeframe: Optional[str] = None
+    last_updated: Optional[int] = None
+    participant_timestamp: Optional[int] = None
+    sip_timestamp: Optional[int] = None
+
+    @staticmethod
+    def from_dict(d):
+        return UniversalSnapshotLastTrade(**d)
+
+
+@modelclass
+class UniversalSnapshotUnderlyingAsset:
+    """Contains data for the underlying stock in an options contract."""
+
+    ticker: Optional[str] = None
+    price: Optional[float] = None
+    value: Optional[float] = None
+    change_to_break_even: Optional[float] = None
+    timeframe: Optional[str] = None
+    last_updated: Optional[int] = None
+
+    @staticmethod
+    def from_dict(d):
+        return UniversalSnapshotUnderlyingAsset(**d)
+
+
+@modelclass
+class UniversalSnapshotDetails:
+    """Contains details for an options contract."""
+
+    contract_type: Optional[str] = None
+    exercise_style: Optional[str] = None
+    expiration_date: Optional[str] = None
+    shares_per_contract: Optional[float] = None
+    strike_price: Optional[float] = None
+
+    @staticmethod
+    def from_dict(d):
+        return UniversalSnapshotDetails(**d)
+
+
+@modelclass
+class UniversalSnapshot:
```

Comment:
Thoughts on this model naming convention? All new models are prefixed with `Universal`, but I didn't create a new model for everything (all but `Greeks`)